Add operator parameter to onERC721Received function.

mg6maciej opened the pull request:
Before approving this PR please verify the correct value of
`bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`

This might be done quickly here: https://emn178.github.io/online-tools/keccak_256.html for `onERC721Received(address,address,uint256,bytes)` and confirm that for previous signature `onERC721Received(address,uint256,bytes)` it returns `f0b9e5ba`.

eip-automerger commented:
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 721 requires approval from one of (nastassia.sachs@protonmail.com, @fulldecent, @dete, @dekz)

fulldecent commented:
I got the value `150b7a02`.

Also, P.S. in your top comment you have an extra space inside the `onERC721Received ...` inside the backticks \`

fulldecent commented:
"triggered `safeTransferFrom` function call" might not be the canonical way to describe this. The Yellow Paper might recommend a different way to say this.

mg6maciej commented:
@fulldecent I have simplified it to "called `safeTransferFrom` function". The Yellow Paper uses a term "message calls", but I'm not convinced this will be easier for Solidity developers to understand.
I couldn't find any extra spaces in the comment above. Might be you copied too much text when verifying value?
I also added minor improvements to the text in this PR: changed `data` to `_data` in `onERC721Received` comments and fixed typo in "transferred". I will send a separate PR with these changes if this PR is rejected.

fulldecent commented on EIPS/eip-721.md:
```diff
@@ -155,22 +155,21 @@ interface ERC165 {
-/// @dev Note: the ERC-165 identifier for this interface is 0xf0b9e5ba
-/// @dev Note: the application will get the prior owner of the token
-///  via _from parameter -- but it will NOT see who called safeTransferFrom.
+/// @dev Note: the ERC-165 identifier for this interface is 0x150b7a02
```
Add a period at end of sentence please. See https://github.com/ethereum/EIPs/pull/1163

fulldecent commented on EIPS/eip-721.md:
```diff
@@ -155,22 +155,21 @@ interface ERC165 {
     ///  transfer. Return of other than the magic value MUST result in the
     ///  transaction being reverted.
     ///  Note: the contract address is always the message sender.
+    /// @param _operator The address which called `safeTransferFrom` function.
```
Remove period, see https://github.com/ethereum/EIPs/pull/1163

mg6maciej commented:
@fulldecent The build passed just a few minutes ago. I fixed punctuation issues and rebased to resolve merge conflicts.

fulldecent commented:
@mg6maciej Thank you, approved, merged.
